### PR TITLE
Shortens title for export formular

### PR DESCRIPTION
**Pull request by thoniTUB**

### CODE REVIEW

File: backend/src/main/java/com/bakdata/conquery/models/config/PreprocessingConfig.java [hunk 1]
```diff
@@ -3,15 +3,13 @@
...
-import org.hibernate.validator.constraints.NotEmpty;
-
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
 
 @Getter @Setter @ToString
 public class PreprocessingConfig {
-	@NotEmpty @Valid
+	@Valid
```
Comment (Kadrian) [inline backend/src/main/java/com/bakdata/conquery/models/config/PreprocessingConfig.java:12]: Hmm is that change intended?

Comment (thoniTUB) [reply backend/src/main/java/com/bakdata/conquery/models/config/PreprocessingConfig.java:12]: Yes, since the preprocess task has been updated to optionally accept these informations as a comand line argument. The Config is then ignored
